internal/topo/transform/template.go

```diff
@@ -70,10 +70,46 @@
 		switch format {
 		case message.FormatJson:
 			if transformed {
```

Comment:
In previous code, we assume after transform, the result is json bytes so that it is return immediately to save one marshal/unmarshal pair. Otherwise, all format can be handled in the same way.